README.md

```diff
@@ -56,6 +60,8 @@ up-cpp, If you are making a project that uses up-cpp, follow the steps in the
 
 ### With Conan for dependencies
 
+Clone the up-cpp repo to a folder named: up-cpp
```

Comment:
That's the default folder name when cloning a repo called up-cpp. Do we really need to document that?

Reply:
The real question is, it is completely clear without it?  The very next instruction says cd to the up-cpp folder with no indication of how that folder was created.  Even if this is the default behavior, I always find it more helpful to be verbose.  Would you rather people unfamiliar w/ git and the default behavior be frustrated and submitting support tickets for what do these instructions mean?  Or just have the instructions be explicit?